app/src/main/res/layout/top_placeholder_item.xml

```diff
@@ -2,6 +2,10 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!-- RecyclerViews will focus the first element that is focusable. This prevents views below
+the top from being scrolled to unintentionally. -->
 <View xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="0dp"/>
+    android:layout_height="0dp"
+    android:focusable="true" />
```

Comment:
would this be a short term stop-gap until the Homepage is all Compose, basically?

Reply:
There was a bit of discussion during this process about how best to handle this in general. One suggestion, for example, was to use an invisible skeleton of roughly the same size as the final content. We could then wait until after first frame to populate the skeleton with content.

I don't know off the top of my head if this issue will reappear when the whole screen is compose. If items in large `LazyList`s are focused automatically we may have to do something similar.